docs/release-notes/release-notes-0.17.1.md

```diff
@@ -0,0 +1,12 @@
+# Release Notes
+
+## Payment
+
+* [Deprecated](https://github.com/lightningnetwork/lnd/pull/7175)
```

Comment:
Can be updated to target 0.70 now (given current state of this PR).

Reply:
Kinda wanna release this with other payment lifecycle-related PRs together tho. Is it possible to merge it without putting it into 0.17.0?

Reply:
Not really, as we're not using a branch (AFAIK) for the 0.17 release. So we can just keep this un-merged until 0.17 is out.